usersimcrs/run_simulation.py

```diff
@@ -41,17 +43,16 @@
 OUTPUT_DIR = "data/runs"
 
 
-def main(config: confuse.Configuration):
+def main(config: confuse.Configuration, agent: Agent) -> None:
     """Executes the specified configuration.
 
     Loads domain and interaction model. Initializes agent and user. Runs the
     simulation.
 
     Args:
         config: Configuration generated from YAML configuration file.
+        agent: Conversational agent.
     """
```

Comment:
What do you think about adding a type check in the beginning for `agent`?

Reply:
Done.